src/groupByKey.test.ts

```diff
@@ -24,10 +24,10 @@ describe('Function "groupByKey"', () => {
     expect(Object.keys(groupedData))
       .toEqual(['22', '23', '24', '26']);
 
-    expect(groupedData[22].map((student) => `${student.name} ${student.age}`))
+    expect(groupedData[22].map((student: { name: any; age: any; }) => `${student.name} ${student.age}`))
```

Comment:
Try to avoid using the type `any`. If it is possible use `string` and `numbre` types.

Reply:
if you want to show that argument is not required use `?`
`interface Object { name?: string, age?: number }`
this sign means that the value is possibly undefined.